### PR TITLE
WinUI-3213 : UG document content for SfMaskedTextBox control

### PR DESCRIPTION
**Pull request by Vignesh-SF3580**
### Description
UG document for SfMaskedTextBox control.

### CODE REVIEW

File: winui-toc.html [hunk 1]
```diff
@@ -359,6 +359,17 @@
             <li><a href="/winui/linear-Gauge/animation">Animation</a></li>
         </ul>
     </li>
+    <li>
+        MaskedTextBox
+        <ul>
+            <li><a href="/winui/MaskedTextBox/Overview">Overview</a>"</li>
```
Comment (Manivannan-E) [inline winui-toc.html:365]: @Vignesh-SF3580 Change the href link with /winui/**Masked-TextBox**/Overview

File: winui/MaskedTextBox/Overview.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+platform: WinUI
+control: SfMaskedTextBox
+documentation: ug
+---
+
+# WinUI MaskedTextBox Overview
+
+The Masked text box is an advanced version of the Entry control that restricts your input to certain types of characters, text, and numbers using a mask pattern. This control is used to create a template for providing information such as telephone numbers, IP addresses, product IDs, and so on.
+
+![MaskedTextBox control overview in WinUI](MaskedTextBox_Images/winui_maskedtextbox_overview.png)
```
Comment (Manivannan-E) [inline winui/MaskedTextBox/Overview.md:14]: @Vignesh-SF3580 Use the below content.

The .NET WinUI SfMaskedTextBox is an advanced version of the input control that restricts input of certain characters, text, and numbers by using a mask pattern. This control is used to create a template for providing information such as telephone numbers, email IDs, IP addresses, product keys, and so on. 

### Key features

* **MaskTypes**: The input can be masked with a fixed or variable length by setting the mask type to simple or regex, respectively.  
* **PromptChar**: Customize the prompt characters used to set the default characters as input in the absence of input. 
* **Value**: Enter values and clipboard operations in the control based on the ValueMaskFormat property.
* **ValueMaskFormat**: Setting formatted values in the control using mask format values such as prompt, literals, and both.
* **Customization**: Customize the UI of the masked text box.

File: winui-toc.html (hunk 1 above)
Comment (Vignesh-SF3580) [reply winui-toc.html]: Changed the href link.

File: winui/MaskedTextBox/Overview.md (hunk 2 above)
Comment (Vignesh-SF3580) [reply winui/MaskedTextBox/Overview.md]: Content changed like above.

File: winui-toc.html [hunk 3]
```diff
@@ -359,6 +359,17 @@
             <li><a href="/winui/linear-Gauge/animation">Animation</a></li>
         </ul>
     </li>
+    <li>
+        Masked TextBox
+        <ul>
+            <li><a href="/winui/Masked-TextBox/overview">Overview</a>"</li>
+            <li><a href="/winui/Masked-TextBox/getting-started">Getting Started</a></li>
+            <li><a href="/winui/Masked-TextBox/masktypes">Mask Types</a></li>
+            <li><a href="/winui/Masked-TextBox/valuemaskformat">Mask Formats</a></li>
```
Comment (Manivannan-E) [inline winui-toc.html:368]: @Vignesh-SF3580 Change like below.

<li><a href="/winui/Masked-TextBox/mask-types">Mask Types</a></li>
<li><a href="/winui/Masked-TextBox/formatting-value">Formatting value</a></li>

Comment (Vignesh-SF3580) [reply winui-toc.html]: Changed like above.

**Review by Manivannan-E — APPROVED** [review]
Changes are fine.

**Pull request merged**